src/Microsoft.Diagnostics.Runtime/src/Builders/RuntimeBuilder.cs

```diff
@@ -1057,7 +1057,7 @@ private void CreateFieldsForMethodTableWorker(ClrType type, out ImmutableArray<C
             statics = staticsBuilder.MoveOrCopyToImmutable();
         }
 
-        public MetaDataImport? GetMetaDataImport(ClrModule module) => _sos.GetMetadataImport(module.Address);
+        public MetaDataImport? GetMetadataImport(ClrModule module) => _sos.GetMetadataImport(module.Address);
```

Comment:
I don't mind following the guidelines you point to as long as it's 100% consistent.  I don't want the class name and the method name to not match.  Note that the reason it's MetaDataImport in ClrMD is because that's how [IMetaDataImport](https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/metadata/imetadataimport-interface) is spelled.

If if you change `MetaData` to `Metadata` I'm fine with that (I like it better even) but also please change the `MetaDataImport` class with this PR too.

Reply:
> Note that the reason it's MetaDataImport in ClrMD is because that's how IMetaDataImport is spelled.

I think the consistency with other types and properties (for example, `MetadataToken`) is more important than to follow the interface's old-style real name. I've renamed that type to adopt the new style.